app/database/accessor/breeze_buddy/knowledge_base.py

```
@@ -0,0 +1,429 @@
+        candidate_k=candidate_k,
+    )
+    try:
+        async for conn in get_db_connection():
+            async with conn.transaction():
+                if _iterative_scan_supported is not False:
+                    try:
+                        await conn.execute(
+                            "SET LOCAL hnsw.iterative_scan = 'relaxed_order'"
+                        )
+                        _iterative_scan_supported = True
+                    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)